apiary.apib

```diff
@@ -7690,11 +7690,12 @@ Updates an existing configuration for [Development Branch](#reference/developmen
 + Response 204 (application/json)
 
 ### Delete Configuration [DELETE /v2/storage/branch/{branch_id}/components/{component_id}/configs/{config_id}]
-Deletes a development branch configuration. When a configuration is deleted, it is only marked as deleted and remains still available in the system.
+Deletes a configuration. When a configuration is deleted, it is only marked as deleted and remains still available in the system.
```

Comment:
```suggestion
Deletes a configuration in specified branch. When a configuration is deleted, it is only marked as deleted and remains still available in the system.
```

Reply:
jakoze nevim proc ti ten `Deletes a development branch configuration` vadil . zni sice divne, ale imho ma pravdu a melo by se to zminit. Nebo ne?